### PR TITLE
Support per-module docker compose files for metricbeat tests

### PR DESCRIPTION
**Pull request by jsoriano**
Modify docker compose helpers so they can start scenarios
using different docker compose files, what will allow to define
docker compose scenarios per module.

Apache service definition is moved to its own docker compose
file.

Related to elastic/beats#12909

### CODE REVIEW

**Comment by jsoriano** [comment]
jenkins, test this again please

**Comment by jsoriano** [comment]
jenkins, test this again please

**Comment by jsoriano** [comment]
jenkins, test this again

**Comment by sayden** [comment]
This is really cool. Looking forward to give a review :+1:

**Comment by jsoriano** [comment]
@sayden thanks! go for it, I have just opened this for review :slightly_smiling_face:

File: libbeat/tests/system/beat/compose.py [hunk 1]
```diff
@@ -51,10 +51,13 @@ def is_healthy(container):
...
 
         project = cls.compose_project()
+        project.build(
+            service_names=cls.COMPOSE_SERVICES,
+            build_args=cls.COMPOSE_BUILD_ARGS)
         project.up(
             strategy=ConvergenceStrategy.always,
             service_names=cls.COMPOSE_SERVICES,
-            do_build=BuildAction.force,
+            do_build=BuildAction.skip,
```
Comment (sayden) [inline libbeat/tests/system/beat/compose.py]: Are you sure about this change?

Comment (jsoriano) [reply libbeat/tests/system/beat/compose.py]: Yes, after adding a explicit `project.build` it is not needed to build also in `project.up`. The reason to add an explicit `project.build` is that  `project.up` doesn't allow to pass build arguments.

**Pull request merged**